python/cudf/cudf/core/indexed_frame.py

```diff
@@ -150,24 +149,14 @@
 )
 
 
-def _get_host_unique(array):
+def _get_unique_drop_labels(array):
```

Comment:
Probably negligible in the context of `.drop`, but it was to avoid a case where `array` was a scalar so we were converting `scalar -> iterable (_get_unique_drop_labels) -> scalar (frame._drop_column(scalar))`. I can change it back to make this `_get_unique_drop_labels` return an iterable if preferred.

Reply:
I’ll leave the choice to you. Just noting that `yield from` patterns tend to be dangerous for performance in cudf because host-device data copying is often involved.